ocfweb/docs/docs/services/hpc.md

```diff
@@ -42,7 +42,7 @@ where you can ask questions and talk to us about anything HPC.
 As of Fall 2018, the OCF HPC cluster is composed of one server, with the
 following specifications:
 
-* 2 Intel Xeon [E5-2640v4][corruption-cpu] CPUs (10c/20t @ 2.4GHz)
+* 2 Intel Xeon [E5-2640v4][corruption-cpu] CPUs (40 cores @ 2.4GHz)
```

Comment:
I'd revert this since nominally it isn't factually accurate - there's only 20 actual cores, 40 threads with SMT.

Reply:
It isn't accurate, but I wanted people to know what the maximum number of CPUs they could allocate in Slurm was.

Reply:
No do not put cores in scare quotes.

Reply:
probably better to make that intention explicit in the same section where you explain how many GPUs can be allocated, rather than in the spec sheet